tests/test_minimal.py

```diff
@@ -23,24 +23,24 @@ def test_minimal_image_size(container, container_runtime):
     """
     if OS_VERSION in ("tumbleweed",):
         minimal_container_max_size: Dict[str, int] = {
-            "aarch64": 63,
-            "ppc64le": 58,
-            "s390x": 41,
-            "x86_64": 49,
+            "aarch64": 51,
+            "ppc64le": 74,
```

Comment:
Everything decreased, except this one. Is this correct?

Reply:
yeah, ppc64 is broken / not publishing anymore, I can't decrease it.